modules/github-repository/variables.tf

```diff
@@ -415,3 +409,80 @@ variable "create_repository" {
   type        = bool
   default     = true
 }
+
+
+variable "branch_name_checker" {
+  description = ""
+  type        = bool
+  default     = false
+}
+
+variable "pr_description_checker" {
+  description = ""
+  type        = bool
+  default     = false
+}
+
+variable "pr_title_checker" {
+  description = ""
+  type        = bool
+  default     = false
+}
+
+variable "pre_commit" {
+  description = ""
+  type        = bool
+  default     = false
+}
+
+variable "semantic_release" {
+  description = ""
+  type        = bool
+  default     = false
+}
+variable "checkov" {
+  description = ""
+  type        = bool
+  default     = false
+}
+
+variable "infracost" {
+  description = ""
+  type        = bool
+  default     = false
+}
+
+variable "terraform-test" {
+  description = ""
+  type        = bool
+  default     = false
+}
+
+variable "tflint" {
+  description = ""
+  type        = bool
+  default     = false
+}
+
+variable "tfsec" {
+  description = ""
+  type        = bool
+  default     = false
+}
+variable "pr_terraform_plan" {
+  description = ""
+  type = object({
+    path_to_module   = string
+    module_variables = map(string)
```

Comment:
why we have path_to_module and module_variables for each actions, cant we have one for all them?